Updated sidebar

dawelsch opened the pull request:
Working sidebar with expand/collapse toggles. Still need to fine tune the toggle behavior, but all pages are showing up and the menu is stable.

shaleman committed:
sidebar changes

dawelsch committed:
Source files only for the sidebar menu structure.

dawelsch committed:
Build of site with new sidebar code.

dawelsch committed:
Fixes to stability of sidebar menu. Clicking sub-items no longer collapses tree.

dawelsch committed:
Merge branch 'master' of https://github.com/pyhung99/pyhung99.github.io
Did build of site, committing updated built files.

dawelsch committed:
Fixed sidebar that broke after last merge

dawelsch committed:
Built fix to sidebar.

dawelsch committed:
Minor changes - added a videos page, fixed sidebar issues.

dawelsch committed:
Build of previous commit.

pyhung99 commented:
Can you do a make build on this before the pull request? I'm seeing more conflicts with this one. Something else in the build changed 1 hour ago also in the websrc directory, so i think this pull is out of date already because it predates the last commit.

dawelsch committed:
Added Networking 101 tutorial to getting_started/network_topology.

dawelsch committed:
Multiple changes to site:
- Final version of landing page (minor changes), including header
- Fix to make sidebar accordion menus expand properly
  - layouts/install.erb
  - assets/stylesheets/_docs.scss (small but important format changes)
- Added embedded video page in Getting Started section
- All documentation pages now either complete or have a stub file for topics

Checking in changes source file only.

dawelsch committed:
Pushing compiled files for last website changes.
